client/payment-details/dispute-details/style.scss

```
@@ -1 +1,7 @@
 /** @format */
+
+.transaction-details-dispute-details-wrapper {
+	margin-left: 24px;
+	margin-right: 24px;
+	margin-bottom: 5px;
```

Part 2 of suggestion to use padding.
```suggestion
	padding-left: 24px;
	padding-right: 24px;
	padding-bottom: 5px;
```